pubspec.yaml

```diff
@@ -6,8 +6,10 @@ environment:
   sdk: '>=2.1.0 <3.0.0'
 
 dependencies:
-  args: "^1.4.0"
-  sass: "^1.17.3"
+  args: "^1.5.1"
+  sass:
+    # Temporary, until 1.17.5 is on pub
+    git: git://github.com/sass/dart-sass
```

Comment:
Use [dependency overrides](https://www.dartlang.org/tools/pub/dependencies#dependency-overrides) for temporary dependencies.